core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -169,6 +170,9 @@
   /** Controls whether state machine can take snapshots. */
   private final AtomicBoolean mSnapshotAllowed;
 
+  private final Map<String, RatisDropwizardExports> mRatisMetricsMap =
```

Comment:
Please add a comment on how this map is utilized by Ratis helpers you implemented.